changelogs/fragments/fix_tests.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - cisco.nxos.nxos_snmp_server - Fix various test cases post nxapi enablement on CML
```

Comment:
nxapi was not enabled in CML rather in the nxos appliance.
Use of agents to generate stuff is fine, but please review things they generate.